manifests/v1alpha3/katib-controller/katib-config.yaml

```diff
@@ -46,5 +46,9 @@ data:
             "memory": "200Mi"
           }
         }
+      },
+      "cmaes": {
+        "image": "gcr.io/kubeflow-images-public/katib/v1alpha3/suggestion-goptuna",
+        "imagePullPolicy": "Always"
```

Comment:
[nits] Most image pull policies of other suggestion service is `IfNotPresent`. So I think you can omit this line.

Reply:
Sure, deleted